data/panel.css

```diff
@@ -59,15 +59,29 @@ video#video {
     background-repeat: no-repeat;
     background-size: cover;
     background: url('img/static.gif');
+    height: 100%;
+}
+
+.error-message-container {
+    background: #00000080;
```

Comment:
I meant to check on this. I basically just copied it out of the css editor. I'll update to something with more support just to be safe